### PR TITLE
rn 0.64.2 upgrade

### PR DESCRIPTION
**Pull request by Inbal-Tish**
TAKE 2

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -6,19 +6,14 @@ platform :ios, '10.0'
...
 end
-
-target 'CalendarsExample-tvOS' do
-  # Pods for CalendarsExample-tvOS
-
-  target 'CalendarsExample-tvOSTests' do
-    inherit! :search_paths
-    # Pods for testing
-  end
-end
```
Comment (lidord-wix) [inline ios/Podfile:24]: In the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2) they're adding the following code as well: 

```
use_flipper!()

post_install do |installer|
  react_native_post_install(installer)
```

**Review by lidord-wix — COMMENTED** [review]
Small comment :)

File: ios/Podfile (hunk 1 above)
Comment (Inbal-Tish) [reply ios/Podfile:24]: I'm not using flipper in calendars...

**Review by lidord-wix — APPROVED** [review]

**Comment by a613** [comment]
https://github.com/wix/react-native-calendars/issues/1530

**Pull request merged**